packages/build/tests/plugins/snapshots/tests.mjs.md

```diff
@@ -2403,9 +2457,7 @@ Generated by [AVA](https://avajs.dev).
     ␊
       Plugin details␊
       Package:        ./plugin.ts␊
-      Version:        1.0.0␊
-      Repository:     git+https://github.com/netlify/build.git␊
-      Report issues:  https://github.com/netlify/build/issues␊
+      Repository:     test␊
```

Comment:
Some of the error messages in the test snapshots change due to the addition of `package.json` in the test fixtures. Those changes are expected.